chapters/syntax.tex

```diff
@@ -26,22 +26,24 @@ \section{Lexical conventions}\doublelabel{lexical-conventions}
          "\a" | "\b" | "\f" | "\n" | "\r" | "\t" | "\v"
 DIGIT = "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
 UNSIGNED-INTEGER = DIGIT { DIGIT }
-UNSIGNED-NUMBER = UNSIGNED-INTEGER [ "." [ UNSIGNED-INTEGER ] ]
-        [ ( "e" | "E" ) [ "+" | "-" ] UNSIGNED-INTEGER ]
+UNSIGNED-REAL = UNSIGNED-INTEGER  "." [ UNSIGNED-INTEGER ]
+     |  UNSIGNED_INTEGER [ "." [ UNSIGNED_INTEGER ] ] ( "e" | "E" ) [ "+" | "-" ] UNSIGNED-INTEGER ]
```

Comment:
It looks like we have an extra closing bracket here.  It could be the reason why some still think it looks as if `13` is a valid `UNSIGNED-REAL`.

Reply:
Yes, that will now be corrected. It must be a rest from splitting off decimals and exponents in two different variants.